### PR TITLE
unifont import,  TCPDF use import fonts

### LINKED ISSUE #18465: daily report does not support unicode characters
when running matomo on websites which have special characters such as japanese or chinese characters, we are not able to get them in the daily report even if we follow https://matomo.org/faq/how-to-install/faq_142/.

## Expected Behavior
Having the characters displayed in the PDF report

## Current Behavior
Current unicode characters are not displayed in the PDF report

## Steps to Reproduce (for Bugs)
Create a webpage with a title in chinese or in japanese.
Check in the report that these characters are well displayed in the page title report.
create a daily report with page titles.
check the generated pdf that the characters for the chinese page are not well displayed.

### PR DESCRIPTION
**Pull request by peterhashair**
### Description:
Fixes: #18465 
if unifont import, use that font for the pdf

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

**Review by bx80 — APPROVED** [review]
Looks good to me, the unifont was detected and applied to email reports :+1:

File: core/ReportRenderer/Pdf.php [hunk 1]
```diff
@@ -136,6 +137,11 @@ public function setLocale($locale)
         }
         // WARNING: Did you read the warning above?
 
+        // if customer fonts import use import font
```
Comment (justinvelluppillai) [inline core/ReportRenderer/Pdf.php]: It'd be good to improve this comment, I'm not sure what it means there

File: core/ReportRenderer/Pdf.php [hunk 2]
```diff
@@ -136,6 +137,11 @@ public function setLocale($locale)
...
+        $fontPath = "plugins/ImageGraph/fonts/unifont.ttf";
+        if (file_exists($fontPath)) {
```
Comment (justinvelluppillai) [inline core/ReportRenderer/Pdf.php]: Consider using `is_file` instead of `file_exists`, see https://github.com/matomo-org/matomo/issues/16552

**Review by Pojielp — COMMENTED** [review]
0546ee3853e24d76bb9a8cf8920f3a40ef6f747d

**Pull request merged**